Moggles/ClientApp/src/featureToggle/EditFeatureToggle.vue

```diff
@@ -78,7 +78,7 @@
             <strong>Change Reasons</strong>
           </h5>
           <hr class="margin-top-1">
-          <textarea v-model="reasonToChange" :disabled="!featureToggleStatusesChanged" class="col-sm-12"
```

Comment:
Is *featureToggleStatusesChanged* still needed? If not, and its purpose was only for disabling/enabling the *reasonToChange* input please also remove all the related code.

Reply:
I removed the unused code after completing a search for anywhere else it may have been used.

Reply:
Removed code that was no longer needed after searching to make sure it wasn't being used elsewhere.